examples/stream-hello-world/StreamHelloWorld_Server.cpp

```diff
@@ -24,7 +24,14 @@ int main(int argc, char* argv[]) {
   // global request handler
   auto handler = std::make_shared<HelloStreamRequestHandler>();
   // start accepting connections
-  rs->start([handler](auto connectionRequest) { return handler; });
+  rs->start([handler](std::unique_ptr<ConnectionSetupRequest> r) {
+    LOG(INFO) << "RSocketServer New Connection => MimeType[Data]: "
```

Comment:
What do you mean by "pretty string formatting"? Do you mean the classes themselves can be output?